### ISSUE TITLE
A11y_ SQLAzureDataStudioMAC_Delete connection_Usability: There is no confirmation dialog for the user to confirm delete operation after the user selects 'Delete connection' menu item.

### ISSUE THREAD

**Issue by bbarik06** [microsoft/azuredatastudio#19869]
**GitHubTags:** #A11yUsable;#A11yTCS;#A11ySev4;#BM_SQLAzureDataStudioMAC_June2022;#SQL Azure Data Studio;#MAC;#FTP;#DesktopApp;#AINotInScope;#A11yUsablehigh;#Keyboard;

## Environment Details:
Application Name: SQL Azure Data Studio
Application Version: Version 1.37.0
Mac Version: macOS Monterey 12.4

## Repro Steps:

1. Please do not close this bug. This bug should only be closed by TCS, C+AI Accessibility tester after verification of the fix.
2. Launch SQL Azure data studio.
3. Connect to a server.
4. Tab till the connected server.
5. Right click on the connected server and select 'Delete Connection'
6. Observe if confirmation dialog appears.

## Actual:
There is no confirmation dialog for the user to confirm delete operation after the user selects 'Delete connection' menu item.

## Expected:
There should be a confirmation dialog for the user to confirm delete operation after the user selects 'Delete connection' menu item.

## User Impact:
Keyboard users will face inconvenience if any important application operation starts getting performed without second level confirmation from the user.

## Attachment:
![2_After clicking del](https://user-images.githubusercontent.com/98393966/176432015-f196c00e-ccd0-4dec-a169-16423d36315c.gif)

**Comment by alanrenmsft** [comment]
bulk closing the usability level issues since we are not planning to work on it anytime soon.
